frontend_tests/node_tests/ui_init.js

```diff
@@ -130,9 +130,18 @@ set_global('$', global.make_zjquery());
 
 const document_stub = $.create('document-stub');
 document.to_$ = () => document_stub;
+document.on_$ = () => document_stub;
 document_stub.on = () => {};
 document_stub.idle = () => {};
 
+zrequire('rows');
+zrequire('Filter', 'js/filter');
+zrequire('FetchStatus', 'js/fetch_status');
```

Comment:
We can just put `message_fetch` into `ignore_modules`, as we discussed over PMs.  Later we should figure out a more standard way to initialize `home_msg_list`, which is kinda hard to reason about and in kind of a strange place (`zulip.js`).

Reply:
OK, switched to that strategy and pushed back here.